NoSoft-SA/nspack
# Extend offload bins trip

Summary of this change:
- add allow_bulk_putaway boolean (default false) to locations table
- add a toggle menu option on the bin_locations grid
- change the existing move_bin(assignment) app to exclude locations with allow_bulk_putaway as true
- add bulk offload to putaway location (validates the bins that were just offloaded from tripsheet,  and moves all the bins to the selected PUTAWAY location)

### Added
- .  allow_bulk_putaway boolean (default false) to locations table
-  a toggle menu option on the bin_locations grid
-  bulk offload to putaway location

### Changed
- . the existing move_bin(assignment) app to exclude locations with allow_bulk_putaway as true

How to test this code via the UI:
Raw Materials | Locations | Storage Definations | Bin Locations
RMD | Raw Material | Offload Bins Trip

* Create a location - `bulk_putaway` should be false.
* Change `bulk_putaway` to true via the action menu. This location should not appear in the list of destination locations when doing move bin.
* From the RMD menu option "Offload Bins Trip", test that all bins on a tripsheet are putaway at the location.
* Only bulk locations should be selectable. Check that the status of the tripsheet and all bins are correct.
* Ensure normal moves are not affected by these changes.
